### DISCUSSION TITLE
ICantBelieveItsNotValetudo addon support for multiple vacuums

### DISCUSSION THREAD

**Discussion by tlpbu** [Poeschl/Hassio-Addons#367] [Ideas]
I have 2 vacuums in houshold. Both running on Valetudo firmware.
Since HA only allows to install single instance of addon is it somehow possible to configure addon already to generate maps for both vaccums?

If not what would it take to implement multi vacuum support?

**Comment by Poeschl** [comment]
There where some thoughts about it a few years ago (#271 #283).

Since the addon only embeds the original ICantBelieveItsNotValetudo an maps the config to a homeassistant compatible config, there is no solution for multiple vacuums right now.

The cleanest solution would be to have the multiple vacuums feature in ICantBelieveItsNotValetudo itself.
